Makefile

```diff
@@ -186,4 +210,17 @@ navigation-fixtures:
 
 .PHONY: 1.0-ui-publish-to-artifactory
 1.0-ui-publish-to-artifactory:
-	./gradlew :libnavigation-ui:artifactoryPublish
\ No newline at end of file
+	./gradlew :libnavigation-ui:artifactoryPublish
+
+.PHONY: 1.0-ui-check-api
+1.0-ui-check-api:
+	# TODO Remove -PhideId=ReferencesHidden after fixing errors
```

Comment:
Currently `> Task :libnavigation-ui:updateApi FAILED`

```
src/main/java/com/mapbox/navigation/ui/NavigationViewModel.java:299: error: Class com.mapbox.navigation.ui.internal.instruction.InstructionModel is hidden but was referenced (as return type parameter) from public method com.mapbox.navigation.ui.NavigationViewModel.retrieveInstructionModel() [ReferencesHidden]
src/main/java/com/mapbox/navigation/ui/NavigationViewModel.java:303: error: Class com.mapbox.navigation.ui.internal.junction.RouteJunctionModel is hidden but was referenced (as return type parameter) from public method com.mapbox.navigation.ui.NavigationViewModel.retrieveRouteJunctionModelUpdates() [ReferencesHidden]
src/main/java/com/mapbox/navigation/ui/NavigationViewModel.java:307: error: Class com.mapbox.navigation.ui.internal.instruction.BannerInstructionModel is hidden but was referenced (as return type parameter) from public method com.mapbox.nsrc/main/java/com/mapbox/navigation/ui/NavigationViewModel.java:307: warning: Method com.mapbox.navigation.ui.NavigationViewModel.retrieveBannerInstructionModel() references hidden type class com.mapbox.navigation.ui.internal.instruction.BannerInstructionModel. [HiddenTypeParameter]
src/main/java/com/mapbox/navigation/ui/NavigationViewModel.java:299: warning: Method com.mapbox.navigation.ui.NavigationViewModel.retrieveInstructionModel() references hidden type class com.mapbox.navigation.ui.internal.instruction.InstructionModel. [HiddenTypeParameter]
src/main/java/com/mapbox/navigation/ui/NavigationViewModel.java:303: warning: Method com.mapbox.navigation.ui.NavigationViewModel.retrieveRouteJunctionModelUpdates() references hidden type class com.mapbox.navigation.ui.internal.junction.
```

We need to address this and remove ☝️ 

cc @abhishek1508